compiler/front/main.nim

```diff
@@ -146,22 +148,27 @@ proc commandCompileToC(graph: ModuleGraph) =
   let conf = graph.config
   extccomp.initVars(conf)
   semanticPasses(graph)
-  if conf.symbolFiles == disabledSf:
+  if conf.symbolFiles == disabledSf and not isDefined(graph.config, "cbackend2"):
```

Comment:
Can be flipped to 
```nim
if isDefined(graph.config, "cbackend2"): ...
elif conf.symbolFiles == disabledSf: ...
```
as you did in line 169.

Reply:
Yep, that's cleaner, thanks.